### PR TITLE
Create LICENSE file so GitHub displays its icon

### PR DESCRIPTION
**Pull request by dandv**
image

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016
```
Comment (connor4312) [inline LICENSE]: Borrowing from Go's LICENSE, this could be:

`Copyright (c) 2016 The node-influx authors`

Comment (dandv) [inline LICENSE]: Updated; please squash the commits if merging :)

**Comment by connor4312** [comment]
Thanks!

**Comment by dandv** [comment]
Thanks as well. BTW, for nicer-looking PR commits:

http://stackoverflow.com/questions/27208042/neat-clean-compacted-git-pr-and-merge-commit-into-one-commit

**Pull request merged**